app/components/code-edit-modal.hbs

```diff
@@ -1,13 +1,23 @@
-<AuModal
-  @modalOpen={{true}}
-  @closeModal={{@onClose}}
->
+<AuModal @modalOpen={{true}} @closeModal={{@onClose}}>
   <:title>Formulier code bewerken</:title>
   <:body>
     <div
-      {{editor @code (perform this.handleCodeChange)}}
+      {{editor this.formCode (perform this.handleCodeChange)}}
       {{!template-lint-disable no-inline-styles}}
       style="overflow: auto; height: 100%; width:100%"
     ></div>
+    <div class="au-u-text-right">
+      <AuButton
+        class="au-u-margin-left-small"
+        @skin="secondary"
+        @disabled={{this.isButtonsDisabled}}
+        {{on "click" this.restoreForm}}
+      >Reset</AuButton>
+      <AuButton
+        class="au-u-margin-left-small"
+        @disabled={{this.isButtonsDisabled}}
+        {{on "click" this.updateForm}}
+      >Bewaar</AuButton>
+    </div>
```

Comment:
Maybe pin these buttons

Reply:
I could fix the styling right away but as we have a meeting in the afternoon on how to make the application more dev focused I will leave this aside for now because the position can change depending on what comes out of the meeting

Reply:
Also mentioned it here on [jira](https://binnenland.atlassian.net/browse/SFB-46?focusedCommentId=91282)